WcaOnRails/config/locales/en.yml

```diff
@@ -371,7 +371,7 @@ en:
       gender:
         male: "Male"
         female: "Female"
-        other: "Other"
+        other_gender: "Other"
```

Comment:
Yes precisely, the key having one of `:zero, :one, :two, :few, :many, :other`, it's considered as being a pluralization key, and `have_valid_pluralization_keys` checks that the other subkeys are valid subkeys (which fails in this case).